setup.py

```diff
@@ -67,36 +67,36 @@
 
 install_requires = [
     # General Invenio dependencies
-    "invenio-app==1.3.0",
-    "invenio-base==1.3.0",
-    "invenio-config==1.0.3",
+    "invenio-app>=1.3.0,<2",
+    "invenio-base>=1.3.0,<2",
+    "invenio-config>=1.0.3,<2",
     # Custom Invenio `base` bundle
-    "invenio-assets==3.0.0",
-    "invenio-accounts==3.0.0",
     "importlib-metadata==4.13.0",
-    "invenio-logging[sentry]==1.3.0",
-    "invenio-rest==1.2.8",
-    "invenio-theme==2.5.7",
+    "invenio-accounts>=3.0.0,<4",
+    "invenio-assets>=3.0.0,<4",
+    "invenio-logging[sentry]==1.3.0", #psaiz NEW MAYOR!!
+    "invenio-rest>=1.2.8,<2",
+    "invenio-theme>=2.5.7,<3",
     # Custom Invenio `metadata` bundle
-    "invenio-indexer==2.1.0",
-    "invenio-jsonschemas==1.1.3",
-    "invenio-pidstore==1.3.1",
+    "invenio-indexer>=2.1.0,<3",
+    "invenio-jsonschemas>=1.1.4,<2",
+    "invenio-pidstore>=1.3.1,<2",
     # This one requires the nested_filter
-    "invenio-records-rest[datacite]==2.4.0",
-    "invenio-records-ui==1.2.0",
-    "invenio-records==2.1.0",
-    "invenio-search-ui==2.8.2",
+    "invenio-records>=2.1.0,<3",
+    "invenio-records-rest[datacite]>=2.4.0,<3",
+    "invenio-records-ui>=1.2.0,<3",
+    "invenio-search-ui>=2.8.3,<3",
     # Custom Invenio `files` bundle
-    "invenio-previewer==2.0.1",
+    "invenio-previewer>=2.0.1,<3",
+    "invenio-records-files>=1.2.1,<3",
     "jupyter-client==7.1.0",
     "pluggy==0.13.1",
-    "invenio-records-files==1.2.1",
     # Custom Invenio `postgresql` bundle
-    "invenio-db[versioning,postgresql]==1.1.0",
+    "invenio-db[versioning,postgresql]>=1.1.0,<2",
     # Custom Invenio `opensearch` bundle
-    "invenio-search[opensearch2]==2.1.0",
+    "invenio-search[opensearch2]>=2.1.0,<3",
     # Specific Invenio dependencies
-    "invenio-xrootd>=1.0.0a6",
+    "invenio-xrootd>=1.0.0a6,<2",
```

Comment:
Maybe because this is an alpha I would be stricter and pin it....wdyt?

Reply:
Yes, sure.